app/controllers/users/invitations_controller.rb

```diff
@@ -0,0 +1,10 @@
+class Users::InvitationsController < Devise::InvitationsController
+  before_action :configure_permitted_parameters
+
+  protected
+
+  def configure_permitted_parameters
+    devise_parameter_sanitizer.permit(:invite, keys: [:user_category])
```

Comment:
I am not sure, but i think we need to add key for email as well

Reply:
email was already added in permit params by device inevitable